src/applications/caregivers/config/chapters/signAsRepresentative/uploadPOADocument.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+// import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
+import { representativeFields } from 'applications/caregivers/definitions/constants';
+import fileUploadUI from 'platform/forms-system/src/js/definitions/file';
+import environment from 'platform/utilities/environment';
+
+// const { representative } = fullSchema.properties;
+// const veteranProps = veteran.properties;
+// const { address, phone } = fullSchema.definitions;
+
+const DocumentUploadDescription = () => {
+  return (
+    <div>
+      <p>
+        If you’re signing as a legal representative, you can upload supporting
+        documents showing your authority to complete this application on behalf
+        of the Veteran.
+      </p>
+
+      <legend style={{ padding: 0 }}>
+        Upload your supporting documentation
+      </legend>
+      <p>
+        You can upload document in a .pdf, .jpeg, or .png file format. You’ll
+        first need to scan a copy of your document onto your computer or mobile
+        phone. You can upload the document from there.
+      </p>
+
+      <p>Guidelines for uploading a file:</p>
+      <ul>
+        <li>File types you. can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 25MB</li>
+      </ul>
+
+      <i>
+        A 1MB file equals about 500 pages of text. A photo is usually about 6MB.
+        Large files can take longer to upload with a slow internet connection.
+      </i>
+    </div>
+  );
+};
+
+const attachmentsSchema = {
+  type: 'array',
+  minItems: 1,
+  items: {
+    type: 'object',
+    required: ['attachmentId', 'name'],
+    properties: {
+      name: {
+        type: 'string',
+      },
+      size: {
+        type: 'integer',
```

Comment:
I didn't know there was an `integer` type... I thought the schemaform only supported `number`.

Reply:
I honestly am not 100% sure copied most of this code over from the HCA do not have the endpoint yet and design wants the UI components done before integrating with the BE so thought I would just come back to it when the endpoint and schema are ready.